clvm/operators.py

```diff
@@ -8,8 +8,9 @@
 from .casts import int_to_bytes
 from .op_utils import operators_for_module
 
+QUOTE_KW = "quote"
```

Comment:
we're changing it from `q` to `quote`?

Reply:
I assume this is to ensure there are no `q`s accidentally lurking anywhere that we forgot to change. The down-side with this is that it makes this patch touch everything, and it makes it harder to interleave into the commit-stream

Reply:
Yes, I think this was the reasoning behind Richard asking for this change.

Reply:
Just talked with Richard. Please disregard this PR until the next update. Thanks!